### PR TITLE
Add README instructions on how to add a new entry

### PR DESCRIPTION
**Pull request by fleupold**
Copying relevant instructions from slack here, so that internal and external contributors can make suggestions (reducing our manual work).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
 # Token Lists
 
-Automativally generate Token Lists
+Automatically generate Token Lists
+
+## How to add an image to the CoW Swap default token list
+
+In order to add a new token to the CoW Swap default token list follow these steps
+
+- Find an icon with reasonable quality/size (~256px, <100kB).
```
Comment (anxolin) [inline README.md:9]: Why not exactly 256x256? I think most of the current images in the list (if not all have that size exactly).

@fairlighteth might have some extra-requirements (or different requirements. What is a good process here?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+- Add the icon to `src/public/images/<chainId>/<address>.png` [^1]
+- Upload the image to Pinata (login in 1Password)
+- Create a new entry in `src/public/CowSwap.json` containing token address, symbol, etc. (**use IPFS link for icon!**)
+- Make sure you bump the version using SemVersions
+- Create a PR with these changes
+
+[^1]: To resize/convert your image [ImageMagick](https://formulae.brew.sh/formula/imagemagick) installed you can simply run `convert ~/Downloads/img.png -resize 256 src/public/images/<chainId>/<address>.png`
```
Comment (anxolin) [inline README.md:16]: Does this optimize like https://tinypng.com do? 
cc/ @fairlighteth

**Review by anxolin — COMMENTED** [review]
Thanks for adding this into the README 🚀

File: README.md [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+
+## How to add an image to the CoW Swap default token list
+
+In order to add a new token to the CoW Swap default token list follow these steps
+
+- Find an icon with reasonable quality/size (~256px, <100kB).
+- Add the icon to `src/public/images/<chainId>/<address>.png` [^1]
+- Upload the image to Pinata (login in 1Password)
+- Create a new entry in `src/public/CowSwap.json` containing token address, symbol, etc. (**use IPFS link for icon!**)
+- Make sure you bump the version using SemVersions
```
Comment (anxolin) [inline README.md:13]: can you link https://github.com/Uniswap/token-lists#semantic-versioning

Comment (anxolin) [reply README.md:13]: Or we can copy the content, whatever your prefer

**Review by alfetopito — APPROVED** [review]

**Review by alfetopito — APPROVED** [review]
LGTM

**Comment by alfetopito** [comment]
Merging as is to have some instructions in the readme.
We can re-iterate if needed, but this is better than no instructions and people not knowing what to do :)

**Pull request merged**